charts/kyverno-policies/Chart.yaml

```diff
@@ -27,8 +27,6 @@ annotations:
       description: Add ability to configure autogen behavior
     - kind: fixed
       description: Support for customLabels, they were ignored up to now
-    - kind: removed
```

Comment:
Please remove the same for the policies chart.

Reply:
@realshuting Done.